src/main/resources/view/ToDoListEventCard.fxml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.CheckBox?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.layout.HBox?>
+
+<HBox prefHeight="25.0" prefWidth="390.0" xmlns="http://javafx.com/javafx/10.0.1" xmlns:fx="http://javafx.com/fxml/1">
+   <children>
+      <CheckBox mnemonicParsing="false" prefHeight="25.0" prefWidth="16.0" />
+      <Label fx:id="id" prefHeight="24.0" prefWidth="25.0" text="$id" />
```

Comment:
might want to control the styles for this in DarkTheme.css 
I was using the hex colors from this site https://elementary.io/brand

Reply:
hopefully we can get it similar to the mockup
https://github.com/CS2103-AY1819S1-T10-1/main/blob/master/docs/images/Ui.png